test/config/test/console-color.test.ts

```diff
@@ -0,0 +1,28 @@
+import { expect, test } from 'vitest'
+import { execa } from 'execa'
+
+// use "execa" directly since "runVitestCli" strips color
+
+test('with color', async () => {
+  const proc = await execa('vitest', ['run', '--root=./fixtures/console-color'], {
+    env: {
+      CI: '1',
+      FORCE_COLOR: '1',
+      NO_COLOR: undefined,
+      GITHUB_ACTIONS: undefined,
```

Comment:
I was puzzled by CI failure and I just found out `GITHUB_ACTIONS` is disabling color:

https://github.com/vitest-dev/vitest/blob/42be8249eac3cadcfaed41f20e2a0147ccb4785d/packages/utils/src/colors.ts#L66-L69

This is a different condition from picocolors https://github.com/alexeyraspopov/picocolors/blob/b6261487e7b81aaab2440e397a356732cad9e342/picocolors.js#L3. For now, I simply unset `GITHUB_ACTIONS` for this test to make it work.